queries/llvm/highlights.scm

```diff
@@ -0,0 +1,12 @@
+(type) @type
+(statement) @statement
+(number) @number
+(comment) @comment
+(string) @string
+(label) @label
+(keyword) @keyword
+(boolean) @boolean
+(float) @number
+(constant) @identifier
+(identifier) @constant
+(punctuation) @punctuation
```

Comment:
I think this causes the CI to fail. We only have punctuation.delimiter

Reply:
Gotcha, should be fixed now!